README.md

```diff
@@ -31,50 +31,53 @@ Ensure you have the shared SDL2 libraries installed on your system first, then t
 > make
 ```
 
-This will create an executable named `./dome` (on Mac OS X and Linux), and `./dome-x32.exe` or `./dome-x64.exe`. 
+This will create an executable named `./dome` (on Mac OS X and Linux), and `./dome-x32.exe` or `./dome-x64.exe`.
 
 ### Run
 
 Run `./dome [gamefile.wren]` to run your game. If your initial file is called `main.wren`, just running `./dome` will execute it. Replace `dome` with your built binary name as necessary.
 
 ## Basics
 
-Your game's entry point must contain a `Game` class which contains at least `static init()`, `static update()` and `static draw(_)` methods.
+Your game's entry point must contain a `Game` class which contains at least `init()`, `update()` and `draw(_)` methods.
```

Comment:
This should say "Your game's entry point must contain a `Game` variable, which holds an object with at least..."